pennylane_forest/qpu.py

```diff
@@ -100,8 +100,9 @@ def __init__(
         calibrate_readout="plus-eig",
         **kwargs,
     ):
+        raise ValueError("Using the QPU via Forest is being deprecated.")
 
-        if readout_error is not None and load_qc:
+        if readout_error is not None and load_qc:  # pylint: disable=unreachable
```

Comment:
Why is this unreachable? :thinking:

Reply:
There is an error that is raised before that